addon/styles/_search.scss

```diff
@@ -0,0 +1,17 @@
+.ui.label.search-container {
+  display: inline-flex;
+  align-items: center;
+  padding: 0%;
+
+  background-color: #fff;
+}
+
+.clear-search {
+  padding: 0.75em 0.75em 0.75em 0.25em;
+  font-size: 1.25em;
+
+  &-disabled {
+    cursor: default;
+    opacity: 0.4;
```

Comment:
It looks like for other places where we have disabled, we use `opacity: 0.5;`. Want to do so here as well?